modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/MultiInstanceUserTaskEventsTest.java

```diff
@@ -179,345 +180,340 @@ public void testMultiInstanceCancelledWhenFlowToTerminateEnd() throws Exception
 
             if ("task2".equals(cancelledEvent.getActivityId())) {
-                assertEquals("task2", cancelledEvent.getActivityId());
-                assertEquals("userTask", cancelledEvent.getActivityType());
-                assertEquals("Multi User Task-${loopCounter}", cancelledEvent.getActivityName());
+                assertThat(cancelledEvent.getActivityId()).isEqualTo("task2");
+                assertThat(cancelledEvent.getActivityType()).isEqualTo("userTask");
+                assertThat(cancelledEvent.getActivityName()).isEqualTo("Multi User Task-${loopCounter}");
                 String eventExecutionId = activityEvent.getExecutionId();
                 if (multiExecutionId1.equals(eventExecutionId)) {
-                    assertEquals(FlowableEngineEventType.ACTIVITY_CANCELLED, activityEvent.getType());
+                    assertThat(activityEvent.getType()).isEqualTo(FlowableEngineEventType.ACTIVITY_CANCELLED);
                     foundMultiExec1 = true;
-                }
-                else if (multiExecutionId2.equals(eventExecutionId)) {
-                    assertEquals(FlowableEngineEventType.ACTIVITY_CANCELLED, activityEvent.getType());
+                } else if (multiExecutionId2.equals(eventExecutionId)) {
+                    assertThat(activityEvent.getType()).isEqualTo(FlowableEngineEventType.ACTIVITY_CANCELLED);
                     foundMultiExec2 = true;
-                }
-                else if (rootMultiExecutionId.equals(eventExecutionId)) {
-                    assertEquals(FlowableEngineEventType.MULTI_INSTANCE_ACTIVITY_CANCELLED, activityEvent.getType());
+                } else if (rootMultiExecutionId.equals(eventExecutionId)) {
+                    assertThat(activityEvent.getType()).isEqualTo(FlowableEngineEventType.MULTI_INSTANCE_ACTIVITY_CANCELLED);
                     foundRootExec = true;
                 }
             } else if ("cancelBoundaryEvent1".equals(cancelledEvent.getActivityId())) {
-                assertEquals(FlowableEngineEventType.ACTIVITY_CANCELLED, activityEvent.getType());
+                assertThat(activityEvent.getType()).isEqualTo(FlowableEngineEventType.ACTIVITY_CANCELLED);
                 cancelledEvent = (FlowableActivityCancelledEvent) activityEvent;
-                assertEquals("cancelBoundaryEvent1", cancelledEvent.getActivityId());
-                assertEquals(boundaryExecution.getId(), activityEvent.getExecutionId());
+                assertThat(cancelledEvent.getActivityId()).isEqualTo("cancelBoundaryEvent1");
+                assertThat(activityEvent.getExecutionId()).isEqualTo(boundaryExecution.getId());
                 foundBoundaryExec = true;
             }
         }
 
-        assert(foundMultiExec1);
-        assert(foundMultiExec2);
-        assert(foundRootExec);
-        assert(foundBoundaryExec);
+        assert (foundMultiExec1);
+        assert (foundMultiExec2);
+        assert (foundRootExec);
+        assert (foundBoundaryExec);
```

Comment:
```suggestion
        assertThat(foundMultiExec1).isTrue();
        assertThat(foundMultiExec2).isTrue();
        assertThat(foundRootExec).isTrue();
        assertThat(foundBoundaryExec).isTrue();
```

Reply:
@filiphr The corrections are made.  The definition of the `assert` methods in the base class that many tests extend makes finding all the old occurrences difficult.

Reply:
Thanks @dbmalkovsky, yes I know. using the java `assert` was most probably some kind of mistake long time ago. This was the only place that I spotted it